tests/phpunit/elementor/includes/template-library/test-manager-remote.php

```diff
@@ -66,4 +100,18 @@ public function test_should_return_wp_error_from_delete_template() {
 		);
 
 	}
+
+    private function mark_template_as_favorite($template_id, $is_favorite): void
+    {
+        $user = $this->factory()->create_and_get_administrator_user();
+        wp_set_current_user($user->ID);
```

Comment:
you can use `$this->act_as_admin()`

Reply:
Maybe put it on the tests and then you can:
```php
$user = $this->act_as_admin();

get_user_meta( $user->ID, //... )

```

Reply:
Done